.drone.yml

```diff
@@ -18273,6 +18196,6 @@ volumes:
   temp: {}
 ---
 kind: signature
-hmac: df00b73f447e6ab65125e08794a80e6ec48bdbfbd6dafdc0449ab2f87c4ecdee
+hmac: f7f0c96cbbabe4305e224c72cd5f9f322b6a54325a890f1537500a5f3eb0e303
```

Comment:
Reminder to double check this before merging.

Reply:
Yep it looks good 👍